### ISSUE TITLE
[Suggestion] Toolbar UI Improvement

### ISSUE THREAD

**Issue by moriwang** [ProxymanApp/Proxyman#1316]
### Proxyman version? (Ex. Proxyman 1.4.3)
Proximal 3.7
### macOS Version? (Ex. mac 10.14)
mac 10.14
### Expected behavior
When app in background, it's too hard for me to know whether debugger tools are active or not. Because the color is grey.

Can we have a small indicator or filled icon?

### Screenshots (optional)
<img width="319" alt="image" src="https://user-images.githubusercontent.com/9151525/182393048-6fa0e7d7-4d8d-48be-a20c-0ae80aff19d6.png">

**Comment by NghiaTranUIT** [comment]
It grays out when the app is in the background is the macOS behavior. It doesn't render the accent color.

If I override it, it would conflict with 

To know which tools are activated, you can check out the bottom bar. It will display the badge if it's active.

![Screen_Shot_2022-08-02_at_21_01_47](https://user-images.githubusercontent.com/5878421/182393935-320f65d9-1659-473a-929a-7c97a5dfb144.jpg)
the macOS behavior.

**Comment by moriwang** [comment]
> It grays out when the app is in the background is the macOS behavior. It doesn't render the accent color.
> 
> If I override it, it would conflict with
> 
> To know which tools are activated, you can check out the bottom bar. It will display the badge if it's active.
> 
> ![Screen_Shot_2022-08-02_at_21_01_47](https://user-images.githubusercontent.com/5878421/182393935-320f65d9-1659-473a-929a-7c97a5dfb144.jpg) the macOS behavior.

Yeah, I know this system behavior, so the suggestion was to change an icon or add some small indicator.
BTW, I found that SSL proxying tools is not in bottom bar.
